PWGGA/GammaConvBase/AliConversionMesonCuts.cxx

```diff
@@ -5047,3 +5073,72 @@ Bool_t AliConversionMesonCuts::IsParticleInJet(std::vector<Double_t> vectorJetEt
 
   return particleInJet;
 }
+
+
+    
+
+///_____________________________________________________________________________
+//   Function to find highest pT track in event
+template <class T, class U>
+Bool_t AliConversionMesonCuts::MesonLeadTrackSelection(T Event, U meson)
```

Comment:
If I understand this correctly you will use this function to select Mesons that are in one of the three areas regarding the leading pT, so that one cut setting will only select a single one?

Reply:
Exactly, thats the idea